flow/flow.yaml

```diff
@@ -69,3 +69,8 @@ components:
         name:
           x-include: ../common/common.yaml#/components/schemas/Named.Object/properties/name
           x-field-uid: 7
+        payload:
+          description: |-
+            The data in the frame after protocol headers and instrumentation bytes.
```

Comment:
I think we can just keep it upto "..after protocol headers." since instrumentation bytes could be in the middle/at the end. 
Maybe : The pattern of the bytes in the transmitted frames for the flow after the protocol headers.